metadata-ingestion/src/datahub/cli/migrate.py

```diff
@@ -141,13 +145,10 @@ def dataplatform2instance_func(
     migration_report = MigrationReport(run_id, dry_run, keep)
     system_metadata = SystemMetadataClass(runId=run_id)
 
-    # initialize for dry-run
-    graph = DataHubGraph(config=DataHubGraphConfig(server="127.0.0.1"))
-
-    if not dry_run:
-        graph = DataHubGraph(
-            config=DataHubGraphConfig(server=cli_utils.get_session_and_host()[1])
-        )
+    if dry_run:
+        graph = DataHubGraph(config=DataHubGraphConfig(server="127.0.0.1"))
+    else:
+        graph = get_default_graph()
```

Comment:
imo it's probably ok to initialize this regardless of the dry run param

we'll regardless need the credentials later on (since cli_utils.get_urns_by_filter will use them), so it's fine to do it here

the 127.0.0.1 thing is just incorrect and we should get rid of it